### ISSUE TITLE
Type problem on mobile

### ISSUE THREAD

**Issue by Arinerron** [valrus/undertale-dialog-generator#61]
Try typing stuff, a newline, type a bit more, then try holding backspace to delete it all on mobile. See what I mean?

**Comment by valrus** [comment]
Yeah, ITDHV is broken as hell on mobile. I haven't even started to look into why.

**Comment by Arinerron** [comment]
I found that if I write it in the URL bar and then select all, copy, then select all in the box and paste, it works fine. I can't seem to type in it though.

**Comment by Arinerron** [comment]
@valrus I don't feel like making a new issue for this, but the sans font is messed up on mobile too. I don't know about pc, haven't tested.
